nowcasting_dataloader/datasets.py

```diff
@@ -179,6 +187,182 @@ def __getitem__(self, batch_idx: int) -> dict:
         return batch
 
 
+class SatFlowDataset(NetCDFDataset):
+    """SatFlow dataset for filtering and splitting up output from NetCDFDataset properly"""
+
+    def __init__(
+        self,
+        n_batches: int,
+        src_path: str,
+        tmp_path: str,
+        configuration: Configuration,
+        cloud: str = "gcp",
```

Comment:
In the future, it'd be good to remove the concept of `cloud` from `nowcasting_dataloader`: #61

(but definitely don't worry about that for this PR!)